docs/source/commandline/commandline-all.rst

```diff
@@ -611,57 +610,57 @@ Core Performance Options
 
 **-[no]autoframeskip** / **-[no]afs**
 
-	Automatically determines the frameskip level while you're running the system, adjusting it constantly in a frantic attempt to keep the system running at full speed. Turning this on overrides the value you have set for -frameskip below. The default is OFF (*-noautoframeskip*).
+	Automatically determines the frameskip level while you're running the system, adjusting it constantly in a frantic attempt to keep the system running at full speed. Turning this on overrides the value you have set for -frameskip below. The default is OFF (``-noautoframeskip``).
 
 .. _mame-commandline-frameskip:
 
 **-frameskip** / **-fs** *<level>*
 
-	Specifies the frameskip value. This is the number of frames out of every 12 to drop when running. For example, if you say -frameskip 2, then MAME will display 10 out of every 12 frames. By skipping those frames, you may be able to get full speed in a system that requires more horsepower than your computer has. The default value is **-frameskip 0**, which skips no frames.
+	Specifies the frameskip value. This is the number of frames out of every 12 to drop when running. For example, if you say ``-frameskip 2``, then MAME will display 10 out of every 12 frames. By skipping those frames, you may be able to get full speed in a system that requires more horsepower than your computer has. The default value is ``-frameskip 0``, which skips no frames.
```

Comment:
These are in command fragment syntax, not INI fragment syntax.  Either make them bold like other command fragments, or change the syntax to INI syntax.  There are a few more cases of this below.